assemblies/assembly-building-custom-dashboard-widgets.adoc

```diff
@@ -1,21 +1,38 @@
 [id='assembly-building-custom-dashboard-widgets']
```

Comment:
It might be worth to change the id because in future, it might not be clear what was "Building custom dashboard widgets"